.golangci.yml

```diff
@@ -0,0 +1,42 @@
+run:
+  timeout: 5m
+  tests: false
+output:
+  formats:
+    - format: github-actions
+    - format: colored-line-number
+linters:
+  disable-all: true
+  enable:
```

Comment:
Can we add the following to the list:
- gocritic
- gosec
- unparam
- errorlint